test/test.js

```diff
@@ -0,0 +1,44 @@
+const TestRunner = require('test-runner');
+const a = require('assert');
+const mock = require('mock-fs');
+const runner = new TestRunner();
+
+runner.test('getByEnv', t => {
+    let config = require('../lib/painless-config');
```

Comment:
Spacing feels off here ;-0

Reply:
tightened it up a bit.
that was a close one 🙈